dame_flame/trim_py.py

```diff
@@ -0,0 +1,29 @@
+import os
```

Comment:
I see this is how you removed the whitespace. We don't want to push this file to the main repo because it's not part of the algorithm. Can you delete it? Thanks!